dev-docs/modules/consentManagement.md

```diff
@@ -0,0 +1,178 @@
+---
+layout: page
+title: Module - GDPR ConsentManagement
+description: User ID persisted in first party domain
```

Comment:
Not sure this is accurate. Maybe change to something like "Add on module to consume and distribute consent information to bidder adapters"

Reply:
this was a hold-over from a page I copied from to start things off that I missed.  Will update it.